src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -41,18 +42,56 @@ export function AccessCodeDetails({
     )
   }
 
+  const filterCodeErrors = () => {
+    return (
+      accessCode?.errors?.filter((error) => {
+        if (error.is_access_code_error) {
+          if (
+            error.error_code === 'failed_to_set_on_device' ||
+            error.error_code === 'failed_to_remove_on_device'
+          ) {
+            return true
+          }
+
+          return false
+        }
+
+        return true
+      }) ?? []
+    )
+  }
```

Comment:
```suggestion
      accessCode?.errors?.filter((error) => {
        if (!error?.is_access_code_error) return true
        if (
          error.error_code === 'failed_to_set_on_device' ||
          error.error_code === 'failed_to_remove_on_device'
        ) {
          return true
        }
        return false
      }) ?? []
    )
  }
```

instead of nesting so many if/else statements, let's use guard statements instead:
https://medium.com/@timothydan/javascript-guard-clauses-64b999e3240